diagramming-tool/src/App.js

```diff
@@ -1,5 +1,15 @@
 import './App.css';
 import Registration from './Component/Register/Registration';
+
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+      <Registration/>
```

Comment:
Registration component should be called in the Routes

Reply:
Done changes